src/main/java/core/mvc/tobe/HandlerMethodArgumentResolverComposite.java

```diff
@@ -15,18 +16,23 @@ public HandlerMethodArgumentResolverComposite(HandlerMethodArgumentResolver... h
 
     @Override
     public boolean supportsParameter(MethodParameter parameter) {
-        return getArgumentResolver(parameter) != null;
+        return getArgumentResolver(parameter).isPresent();
     }
 
     @Override
     public Object resolveArgument(MethodParameter parameter, HttpServletRequest request) {
-        return getArgumentResolver(parameter).resolveArgument(parameter, request);
+        Optional<HandlerMethodArgumentResolver> resolverOptional = getArgumentResolver(parameter);
+
+        if(resolverOptional.isPresent()) {
+            return resolverOptional.get().resolveArgument(parameter, request);
+        }
+
+        return null;
```

Comment:
Optional 을 아래와 같이 활용해도 좋을 것 같아요 😃 
```suggestion
        return getArgumentResolver(parameter)
                .map(handlerMethodArgumentResolver -> handlerMethodArgumentResolver.resolveArgument(parameter, request))
                .orElse(null);
```